src/pages/choose-stacking-method/hooks/index.ts

```diff
@@ -60,3 +60,17 @@ export const useLiquidStackingButton = (props: ChooseStackingMethodLayoutProps)
 
   return { isDisabled, onClick };
 };
+
+const leatherDetected = window.LeatherProvider !== undefined;
+
+export const useLeatherSBTCBridgeButton = () => {
```

Comment:
define as fn, and consider sbtc a word to avoid `SBTCB` capitalisation
```suggestion
export function useLeatherSbtcBridgeButton() {
```